pmm-app/tests/pages/dashboardPage.js

```diff
@@ -321,19 +321,19 @@ module.exports = {
   },
 
   async verifyThereAreNoGraphsWithNA(acceptableNACount = 0) {
-    const numberOfNAElements = await I.grabNumberOfVisibleElements(this.fields.notAvailableMetrics);
+    let numberOfNAElements = await I.grabNumberOfVisibleElements(this.fields.notAvailableMetrics);
```

Comment:
I see no reason they should not be `const`. If IDE complains, then you'd probably want to tune it up.

Reply:
Done thanks